stable/jenkins/templates/jenkins-master-route.yaml

```diff
@@ -29,4 +30,6 @@ spec:
     name: {{ template "jenkins.fullname" . }}
     weight: 100
   wildcardPolicy: None
+status:
+  ingress: []
```

Comment:
We should not include a status in the generated manifests.

Reply:
Adding `status` is needed to solve the problem that this PR is trying to address, that is being able to create this route using Helm 3. You can see that in the error I pasted and in the issue linked in the PR description.

Reply:
My feeling is that we would fix it at the wrong location. For me it's rather an upstream issue that those fields are marked as required. Introducing a fix here would also mean to remove it later.

Reply:
While I completely agree that the status should have been made optional in the spec. As it stands, I'd argue that this chart is wrong by creating a resource that doesn't follow its specification.